gluster/file/README.md

```diff
@@ -61,43 +53,43 @@ parameters:
     snapfactor: "10"
 ```
 
-* `resturl` : Gluster REST service/Heketi service url which provision gluster File volumes on demand. The general format should be `IPaddress:Port` and this is a mandatory parameter for glusterfile dynamic provisioner. If Heketi service is exposed as a routable service in openshift/kubernetes setup, this can have a format similar to
+* `resturl` : heketi service url which provision GlusterFS volumes on demand. The general format should be `IPaddress:Port` and this is a mandatory parameter for the provisioner. If Heketi service is exposed as a routable service in openshift/kubernetes setup, this can have a format similar to
```

Comment:
Lets keep the string `Gluster REST service` in OR form with `heketi`. There is a reason behind it.

Reply:
If you're thinking ahead to gd2, I'd rather leave it as just heketi and then when we're ready change it to `glusterd2/heketi`.